src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -76,8 +76,8 @@ public PluginManager(Jenkins jenkins) {
      * Force update the plugin update center metadata.
      */
     public void checkForUpdates() {
-        visit("checkUpdates");
-        waitFor(by.xpath("//span[@id='completionMarker' and text()='Done']"));
+        visit("advanced");
+        clickButton("Check now");
```

Comment:
Why do you no longer wait for the check to happen - it may not be instant - and smells like a :bug: